pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -886,46 +878,10 @@ func validateLaunchSecurity(field *k8sfield.Path, spec *v1.VirtualMachineInstanc
 	return causes
 }
 
-func appendStatusCauseForPodNetworkDefinedWithMultusDefaultNetworkDefined(field *k8sfield.Path, causes []metav1.StatusCause) []metav1.StatusCause {
-	return append(causes, metav1.StatusCause{
-		Type:    metav1.CauseTypeFieldValueInvalid,
-		Message: "Pod network cannot be defined when Multus default network is defined",
-		Field:   field.Child("networks").String(),
-	})
-}
-
-func appendStatusCaseForMoreThanOneMultusDefaultNetwork(field *k8sfield.Path, causes []metav1.StatusCause) []metav1.StatusCause {
-	return append(causes, metav1.StatusCause{
-		Type:    metav1.CauseTypeFieldValueInvalid,
-		Message: "Multus CNI should only have one default network",
-		Field:   field.Child("networks").String(),
-	})
-}
-
-func validateNetworks(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec, networkNameMap map[string]*v1.Network) (podExists bool, multusDefaultCount int, causes []metav1.StatusCause) {
-
-	podExists = false
-	multusDefaultCount = 0
-
-	for idx, network := range spec.Networks {
-
-		cniTypesCount := 0
-
-		if network.Pod != nil {
-			cniTypesCount++
-			podExists = true
-		}
-
-		if network.NetworkSource.Multus != nil {
-			cniTypesCount++
-			if network.NetworkSource.Multus.Default {
-				multusDefaultCount++
-			}
-		}
-
+func validateNetworks(spec *v1.VirtualMachineInstanceSpec, networkNameMap map[string]*v1.Network) {
```

Comment:
It would be easier to understand if the commit message said this function is removed in follow up commit.

Reply:
Why? The remaining functionality for indexing is still needed after moving the validations. Other validation uses it.